pyseir/inference/model_fitter.py

```diff
@@ -45,12 +47,13 @@ class ModelFitter:
         systematic errors of cases by this number. Thus higher numbers reduce
         the influence of hospitalization data on the fit.
     """
+
     def __init__(self,
                  fips,
                  ref_date=datetime(year=2020, month=1, day=1),
                  min_deaths=2,
                  n_years=1,
-                 cases_to_deaths_err_factor=1,
+                 cases_to_deaths_err_factor=.333,
```

Comment:
How is this determined? Did you test different ratios and find best fit at this ratio?

Reply:
Yes. Later we will be able to reduce this, but for now we are seeing that death data is noisy for so many states (like 5 deaths) that we need more weight in the leading indicators like cases to effectively calculate R0.

Reply:
Cool! Thanks for the clarification!

Reply:
Would better to put add a single grid search at state level for the best ratio as we have more death data surfaced.

Reply:
I am not sure what the optimization objective would be. Goodness of fit doesn't make sense I don't think.